web/js/palettes/model.js

```diff
@@ -479,12 +480,13 @@ export function palettesModel(models, config) {
   };
 
   var updateLookup = function (layerId) {
+    var legend, oldLegend, entries;
+    var lookup = {};
```

Comment:
You're moving this declaration before the ["early return"](https://forum.freecodecamp.org/t/the-return-early-pattern/19364) on the next line. There would be no need to declare it if the function returns early.